DESCRIPTION

```diff
@@ -60,9 +60,9 @@ Suggests:
     testthis,
     withr
 Remotes:
-    Infectious-Disease-Modeling-Hubs/hubUtils,
-    Infectious-Disease-Modeling-Hubs/hubData,
-    Infectious-Disease-Modeling-Hubs/hubAdmin,
+    Infectious-Disease-Modeling-Hubs/hubUtils@enhancement/v3-utils,
+    Infectious-Disease-Modeling-Hubs/hubData@feature/handle-samples,
+    Infectious-Disease-Modeling-Hubs/hubAdmin@feature/sample-support,
```

Comment:
noting that we will probably want to change this before going ahead with a merge

Reply:
Most definitely.
It's the last thing that needs to be done throughout the packages. It is required atm for tests to run successfully.